rest_framework_simplejwt/token_blacklist/models.py

```diff
@@ -20,8 +22,10 @@ class Meta:
         #
         # Also see corresponding ticket:
         # https://github.com/encode/django-rest-framework/issues/705
+        # If we are TYPE_CHECKING, we don't want it to be abstract so django-stubs can pick it up
```

Comment:
The mypy django-stubs plugin does some "magic" behind the scenes to infer types from models and managers (for example, defining `jti = models.CharField(unique=True, max_length=255)` would seem that `OutstandingToken.jti` is of the type `Charfield`; but Django "magic" makes it so acessing the attribute returns a python string instead. django-stubs doesn't seem to pick up on this model if it's defined as "abstract = settings <conditional>" (because it can't quite know if it's abstract or not) and will have some weird behaviour. Adding the `and not TYPE_CHECKING` conditional makes it so it will always evaluate to `False` while typechecking, making django-stubs work properly.